tensornetwork/backends/tensorflow/tensorflow_backend.py

```diff
@@ -58,29 +58,29 @@ def slice(self, tensor: Tensor, start_indices: Tuple[int, ...],
                        "identical.")
     return tf.slice(tensor, start_indices, slice_sizes)
 
-  def svd_decomposition(
+  def svd(
       self,
       tensor: Tensor,
-      split_axis: int,
+      pivot_axis: int = 1,
```

Comment:
Is this consistent default? I assumed it would split on the last axis.

Reply:
It's so that pivot is a null-op on matrices: 
A = np.ones((4, 2))
B = A.reshape( (A.shape[:1], A.shape[1:]. ) ) # new shape is also (4, 2)

Reply:
-1 would also behave this way on matrices, if you prefer that

Reply:
I think i'm slightly more in favor of -1.

Reply:
Okay, I changed it to -1. I think that's a bit better too on reflection